### PR TITLE
NAM-168 -- ENG: Refactor Student Images to work with the official flow

### PR DESCRIPTION
**Pull request by maseca**
# Description
> If no likeness alpha
If no alpha official
If no official likeness default
Making the carousel a static order (likeness alpha official)
***
# Testing
* Verify carousel is displaying in correct order
* Verify image_priority is a string, not an array
* Note:  Uploading needs to be fixed in a later task

### CODE REVIEW

**Review by liadGolan — CHANGES_REQUESTED** [review]
Files are being tracked that should not be and need to get untracked before approval (looks like permission getting tracked)

**Review by liadGolan — APPROVED** [review]
:+1:

**Review by chippingcolt — APPROVED** [review]
Looks Good! Tested by your criteria and it works

**Pull request merged**
